WordPress/src/main/res/drawable/ic_cloud_off_white_24dp.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24">
+  <path
+      android:pathData="M19.35,10.04C18.67,6.59 15.64,4 12,4c-1.48,0 -2.85,0.43 -4.01,1.17l1.46,1.46C10.21,6.23 11.08,6 12,6c3.04,0 5.5,2.46 5.5,5.5v0.5H19c1.66,0 3,1.34 3,3 0,1.13 -0.64,2.11 -1.56,2.62l1.45,1.45C23.16,18.16 24,16.68 24,15c0,-2.64 -2.05,-4.78 -4.65,-4.96zM3,5.27l2.75,2.74C2.56,8.15 0,10.77 0,14c0,3.31 2.69,6 6,6h11.73l2,2L21,20.73 4.27,4 3,5.27zM7.73,10l8,8H6c-2.21,0 -4,-1.79 -4,-4s1.79,-4 4,-4h1.73z"
+      android:fillColor="#ffffff"/>
```

Comment:
🔍 Nitpick: We might want to consider replacing this with a color reference. Same applies to the other files.

Reply:
You are 100% correct ... in commit 5f1e851

Reply:
👍 Thanks!

Just in case you missed, there are still two files ic_gridicons_checkmark_circle.xml and ic_query_builder_white_24dp.xml which have hardcoded values. We can update them later.

Reply:
I updated in commit ad74918. Simple enough to do it now.